dev/com.ibm.ws.jdbc_fat_krb5/fat/src/com/ibm/ws/jdbc/fat/krb5/containers/DB2KerberosContainer.java

```diff
@@ -13,39 +13,24 @@
 import static com.ibm.ws.jdbc.fat.krb5.containers.KerberosContainer.KRB5_KDC;
 import static com.ibm.ws.jdbc.fat.krb5.containers.KerberosContainer.KRB5_REALM;
 
-import java.io.FileInputStream;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.nio.file.Files;
-import java.nio.file.Path;
-import java.nio.file.Paths;
 import java.time.Duration;
-import java.util.Properties;
 
 import org.testcontainers.containers.Db2Container;
 import org.testcontainers.containers.Network;
-import org.testcontainers.containers.output.OutputFrame;
 import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
 import org.testcontainers.utility.DockerImageName;
 
-import com.ibm.websphere.simplicity.log.Log;
-import com.ibm.ws.jdbc.fat.krb5.FATSuite;
-
+import componenttest.containers.SimpleLogConsumer;
 import componenttest.custom.junit.runner.FATRunner;
 
 public class DB2KerberosContainer extends Db2Container {
 
     private static final Class<?> c = DB2KerberosContainer.class;
-    private static final Path reuseCache = Paths.get("..", "..", "cache", "db2.properties");
     // NOTE: If this is ever updated, don't forget to push to docker hub, but DO NOT overwrite existing versions
     private static final String IMAGE = "aguibert/krb5-db2:1.0";
     private static final DockerImageName db2Image = DockerImageName.parse(IMAGE)
```

Comment:
Similar to previous comment.  Consider externalizing these two values (via public) so they can be used instead of hardcoded values in other classes.

Reply:
In this case the `krb5-db2:1.0` image is specific to these Kerberos tests and it wouldn't make sense to use this image elsewhere.